### ISSUE TITLE
[LOC] Support alt text based fragments

### ISSUE THREAD

**Issue by auniverseaway** [adobe/da-nx#44]
As an author, I would like DA Loc to find fragments within the alt text of images.

**Comment by auniverseaway** [comment]
I think this is a little more critical because of the lack of visibility of videos in alt text.

**Comment by chrischrischris** [comment]
related slack convo: https://adobe-mwp.slack.com/archives/C07UKLSNPAA/p1752759775212139
